slices/apache2.yaml

```diff
@@ -0,0 +1,316 @@
+package: apache2
+
+essential:
+  - apache2_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2-bin_bins
+      - apache2_configs
+      - apache2-data_data
+      - apache2-utils_bins
+      - media-types_data
+      - perl_bins
```

Comment:
Instead of depending on `perl_bins`, I think we should instead depend on `perl-base_bins` & `perl-base_modules`.